app/styles/typography/_typescale.scss

```diff
@@ -0,0 +1,36 @@
+// These will live in design system after we ensure everything looks good in mobile-wiki
+.wds-text-2xs {
+	font-size: 10px;
```

Comment:
hmm, maybe we should use rems here?
so if users zoom in, whole page zoom in?

Reply:
mobile-wiki does not allow to zoom, does other apps allow?